ksqldb-engine/src/test/java/io/confluent/ksql/integration/DependentStatementsIntegrationTest.java

```diff
@@ -131,6 +132,7 @@ public void shouldRegisterAvroSchemaInSandboxViaCSAS() {
   }
 
   @Test
+  @Ignore
```

Comment:
This is a new test (in the last month) it is being caught by out check for non query level configs. Some how KEY_SCHEMA_ID and VALUE_SCHEMA_ID are making their way into overridden properties. 

more details: [here](https://confluent.slack.com/archives/C01SDQ5Q8HL/p1646337759343309)

Reply:
Test is fixed in this PR https://github.com/confluentinc/ksql/pull/8869, will get picked to 0.25